app/gas/management/commands/output_gas_viz.py

```diff
@@ -0,0 +1,142 @@
+import datetime
+import os
+import random
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from gas.models import GasProfile
+from numpy import array
+
+
+def convert_to_movie():
+    command = "ffmpeg -framerate 30 -pattern_type glob -i 'cache/frames/*.jpg' -c:v libx264 -pix_fmt yuv420p cache/out.mp4"
+    print("converting to movie")
+    os.system(command)
+
+
+def install_ffmpeg():
+    command = 'apt update -y '
+    os.system(command)
+    command = 'apt install ffmpeg libav-tools x264 x265 -y'
```

Comment:
Can we add `ffmpeg libav-tools x264 x265` to the dockerfile build steps?

Reply:
updated